# Removes in order:
# 'Tabs from other devices' entry on the History page sidebar
# the 'Learn more' link on new incognito tabs
# Live captions entry from the settings page
# link to Google's accessibility site from the settings page
# link to Google's help site on the About page
# external link for theme entry on settings page
# Check passwords and managed passwords sections from the passwords page
# Safety check section on the settings page
# Google sign-in and Anonymized Data Collection sections
# Advanced Protection Program link on the security settings page
# the 'Learn more' link from the search engine entry on the settings page
# Safety Check entry on the side menu on the settings page
# the (?) learn more button on many settings pages
# unneeded elements from the profile menu

--- a/chrome/browser/resources/history/side_bar.html
+++ b/chrome/browser/resources/history/side_bar.html
@@ -111,11 +111,6 @@
             <paper-ripple></paper-ripple>
           </a>
         </template>
-        <a id="syncedTabs" role="menuitem" href="/syncedTabs" class="page-item"
-            path="syncedTabs" on-click="onItemClick_">
-          $i18n{openTabsMenuItem}
-          <paper-ripple></paper-ripple>
-        </a>
         <div class="separator"></div>
         <a role="menuitem" id="clear-browsing-data"
             href="chrome://settings/clearBrowserData"
--- a/chrome/browser/resources/ntp4/incognito_tab.html
+++ b/chrome/browser/resources/ntp4/incognito_tab.html
@@ -24,8 +24,6 @@ document.querySelector('#incognitothemec
   <h1>$i18n{incognitoTabHeading}</h1>
   <p id="subtitle">
     <span>$i18n{incognitoTabDescription}</span>
-    <a class="learn-more-button"
-        href="$i18n{learnMoreLink}">$i18n{learnMore}</a>
   </p>
   <div id="bulletpoints-wrapper">
     <div class="bulletpoints first">$i18nRaw{incognitoTabFeatures}</div>
--- a/chrome/browser/resources/settings/a11y_page/a11y_page.html
+++ b/chrome/browser/resources/settings/a11y_page/a11y_page.html
@@ -21,7 +21,7 @@
               external$="[[captionSettingsOpensExternally_]]">
           </cr-link-row>
         </template>
-        <template is="dom-if" if="[[!captionSettingsOpensExternally_]]">
+        <template is="dom-if" if="[[false]]">
           <cr-link-row id="captions"
               label="$i18n{captionsTitle}"
               on-click="onCaptionsClick_"
@@ -51,10 +51,6 @@
             sub-label="$i18n{accessibleImageLabelsSubtitle}">
         </settings-toggle-button>
 </if>
-        <cr-link-row class="hr" label="$i18n{moreFeaturesLink}"
-            on-click="onMoreFeaturesLinkClick_" sub-label="$i18n{a11yWebStore}"
-            external>
-        </cr-link-row>
       </div>
 <if expr="not chromeos">
       <template is="dom-if" if="[[!captionSettingsOpensExternally_]]">
--- a/chrome/browser/resources/settings/about_page/about_page.html
+++ b/chrome/browser/resources/settings/about_page/about_page.html
@@ -131,8 +131,6 @@
         </div>
       </template>
 </if>
-      <cr-link-row class="hr" id="help" on-click="onHelpTap_"
-          label="$i18n{aboutGetHelpUsingChrome}" external></cr-link-row>
 <if expr="_google_chrome">
       <cr-link-row class="hr" id="reportIssue" on-click="onReportIssueTap_"
           hidden="[[!prefs.feedback_allowed.value]]"
--- a/chrome/browser/resources/settings/appearance_page/appearance_page.html
+++ b/chrome/browser/resources/settings/appearance_page/appearance_page.html
@@ -22,11 +22,11 @@
     <settings-animated-pages id="pages" section="appearance"
         focus-config="[[focusConfig_]]">
       <div route-path="default">
-        <div class="settings-row first" id="themeRow"
+        <div class="cr-row first" id="themeRow"
             hidden="[[!pageVisibility.setTheme]]">
-          <cr-link-row class="first" hidden="[[!pageVisibility.setTheme]]"
-              label="$i18n{themes}" sub-label="[[themeSublabel_]]"
-              on-click="openThemeUrl_" external></cr-link-row>
+          <div class="flex cr-padded-text">
+            <div>$i18n{themes}</div><div class="secondary">[[themeSublabel_]]</div>
+          </div>
 <if expr="not is_linux or chromeos or lacros">
           <template is="dom-if" if="[[prefs.extensions.theme.id.value]]">
             <div class="separator"></div>
--- a/chrome/browser/resources/settings/autofill_page/passwords_section.html
+++ b/chrome/browser/resources/settings/autofill_page/passwords_section.html
@@ -106,6 +106,7 @@
         label="$i18n{trustedVaultOptInLabel}"
         sub-label="$i18n{trustedVaultOptInSubLabel}">
     </cr-link-row>
+<if expr="false">
     <div id="checkPasswordsBannerContainer" class="cr-row"
         hidden$="[[!shouldShowBanner_]]">
       <picture>
@@ -154,6 +155,7 @@
       <!-- This div lays out the link correctly, relative to the text. -->
       <div class="cr-padded-text">$i18nRaw{managePasswordsLabel}</div>
     </div>
+</if>
     <div class="cr-row first">
       <h2 id="savedPasswordsHeading" class="flex">
         $i18n{savedPasswordsHeading}
--- a/chrome/browser/resources/settings/basic_page/basic_page.html
+++ b/chrome/browser/resources/settings/basic_page/basic_page.html
@@ -102,7 +102,7 @@
             <settings-privacy-review-promo></settings-privacy-review-promo>
           </settings-section>
         </template>
-        <template is="dom-if" if="[[showPage_(pageVisibility.safetyCheck)]]"
+        <template is="dom-if" if="[[false]]"
             restamp>
           <settings-section page-title="$i18n{safetyCheckSectionTitle}"
               section="safetyCheck" nest-under-section="privacy"
--- a/chrome/browser/resources/settings/privacy_page/personalization_options.html
+++ b/chrome/browser/resources/settings/privacy_page/personalization_options.html
@@ -8,7 +8,7 @@
         border-top: none;
       }
     </style>
-<if expr="not chromeos">
+<if expr="false">
     <settings-toggle-button id="signinAllowedToggle"
         class="hr"
         hidden="[[!signinAvailable_]]"
@@ -51,12 +51,6 @@
     </settings-toggle-button>
 </if><!-- not chromeos -->
 </if><!-- _google_chrome -->
-    <settings-toggle-button id="urlCollectionToggle"
-        class="hr"
-        pref="{{prefs.url_keyed_anonymized_data_collection.enabled}}"
-        label="$i18n{urlKeyedAnonymizedDataCollection}"
-        sub-label="$i18n{urlKeyedAnonymizedDataCollectionDesc}">
-    </settings-toggle-button>
 <if expr="_google_chrome">
     <settings-toggle-button id="spellCheckControl"
         class="hr"
--- a/chrome/browser/resources/settings/privacy_page/security_page.html
+++ b/chrome/browser/resources/settings/privacy_page/security_page.html
@@ -96,10 +96,3 @@
         sub-label="$i18n{manageCertificatesDescription}"
         on-click="onManageCertificatesClick_"></cr-link-row>
 </if>
-    <cr-link-row id="advanced-protection-program-link"
-        class="hr"
-        label="$i18n{advancedProtectionProgramTitle}"
-        sub-label="$i18n{advancedProtectionProgramDesc}"
-        on-click="onAdvancedProtectionProgramLinkClick_"
-        external>
-    </cr-link-row>
--- a/chrome/browser/resources/settings/search_page/search_page.html
+++ b/chrome/browser/resources/settings/search_page/search_page.html
@@ -12,10 +12,6 @@
     <div class="cr-row first">
       <div id="searchExplanation" class="flex cr-padded-text">
         $i18n{searchExplanation}
-        <a href="$i18n{searchExplanationLearnMoreURL}"
-            target="_blank">
-          $i18n{learnMore}
-        </a>
       </div>
       <template is="dom-if" if="[[isDefaultSearchControlledByPolicy_(
           prefs.default_search_provider_data.template_url_data)]]">
--- a/chrome/browser/resources/settings/settings_menu/settings_menu.html
+++ b/chrome/browser/resources/settings/settings_menu/settings_menu.html
@@ -111,7 +111,7 @@
             <iron-icon icon="settings:assignment"></iron-icon>
             $i18n{autofillPageTitle}
           </a>
-          <template is="dom-if" if="[[!enableLandingPageRedesign_]]">
+          <template is="dom-if" if="[[false]]">
             <a role="menuitem" href="/safetyCheck"
                 hidden="[[!pageVisibility.safetyCheck]]"
                 id="safetyCheck">
--- a/chrome/browser/resources/settings/settings_page/settings_subpage.html
+++ b/chrome/browser/resources/settings/settings_page/settings_subpage.html
@@ -72,7 +72,7 @@
       </template>
       <h1 class="cr-title-text">[[pageTitle]]</h1>
       <slot name="subpage-title-extra"></slot>
-      <template is="dom-if" if="[[learnMoreUrl]]">
+      <template is="dom-if" if="[[false]]">
         <cr-icon-button iron-icon="cr:help-outline" dir="ltr"
             aria-label="$i18n{learnMore}" on-click="onHelpClick_">
         </cr-icon-button>
--- a/chrome/browser/ui/views/profiles/profile_menu_view.cc
+++ b/chrome/browser/ui/views/profiles/profile_menu_view.cc
@@ -175,6 +175,7 @@ void ProfileMenuView::BuildMenu() {
 }
 
 gfx::ImageSkia ProfileMenuView::GetSyncIcon() const {
+  return gfx::ImageSkia();
   Profile* profile = browser()->profile();
   if (profile->IsOffTheRecord() || profile->IsGuestSession())
     return gfx::ImageSkia();
@@ -450,8 +451,7 @@ void ProfileMenuView::BuildIdentity() {
         menu_subtitle_);
   } else {
     menu_title_ = std::u16string();
-    menu_subtitle_ =
-        l10n_util::GetStringUTF16(IDS_PROFILES_LOCAL_PROFILE_STATE);
+    menu_subtitle_ = std::u16string();
     SetProfileIdentityInfo(
         profile_name, background_color, edit_button_params,
         ui::ImageModel::FromImage(
